packages/salesforcedx-vscode-core/src/conflict/conflictUtils.ts

```diff
@@ -20,7 +20,9 @@ import { DeployRetrieveOperationType } from '../util/types';
 import { diffComponents } from './componentDiffer';
 import { DirectoryDiffResults, TimestampFileProperties } from './directoryDiffer';
 import { getConflictMessagesFor } from './messages';
-import { MetadataCacheService } from './metadataCacheService';
+import { MetadataCacheResult, MetadataCacheService } from './metadataCacheService';
+
+type Conflict = Awaited<ReturnType<typeof SourceTracking.prototype.getConflicts>>[number];
```

Comment:
type for a single Conflict from tracking.  All the calls to these were using `getConflicts` but these are typed `any`.  You'll see further down why this comes in handy!